src/views/active.js

```diff
@@ -114,6 +137,59 @@ export const Active = () => {
                     </Fragment>
                 );
             },
+       }, {
+            key: "action",
+            text: "Action",
+            className: "action",
+            width: 100,
+            center: true,
+            sortable: false,
+            name: "Update",
+            cell: (record) => {
+                return (
+                    <Fragment>
+                        <button type="button" value="update" onClick={(e) => handleModalOpen(e, record.sid)}>Update</button>
+                        {modalOpen ?
+                            <Modal>
+                                <Modal.Content>
+                                    <Modal.FormGroup>
+                                        <Modal.FormButton style={{top: "20px"}} type="button" className="close" onClick={() => setModalOpen(false)}>&times;</Modal.FormButton>
+                                        <Modal.FormLabel>Workspace Name</Modal.FormLabel>
+                                        <Modal.FormInput
+                                            ref={workspaceN}
+                                            type="text"
+                                            name="workspace_name"
+                                            placeholder="work instance 1"
+                                        />
+                                        <Modal.FormLabel key="formKey">CPU in cores</Modal.FormLabel>
+                                        <Modal.FormInput
+                                            ref={cpu}
+                                            type="text"
+                                            name="cpu"
+                                            placeholder="1, 2...."
+                                        />
+                                        <Modal.FormLabel key="formKey">Memory in Mi</Modal.FormLabel>
+                                        <Modal.FormInput
+                                            ref={memory}
+                                            type="text"
+                                            name="memory"
+                                            placeholder="1000, 2500...."
+                                        />
```

Comment:
Do we need to validate the setting input value? My understanding is the updated setting should stay in the range of min~max resource provided in the app card. Otherwise we might receive an error from tycho.

Reply:
Yes, we need to validate those inputs. I left those validations for now since I am not entirely sure on how to implement that. The re-naming part works as expected.